mautrix_facebook/example-config.yaml

```
@@ -257,6 +257,8 @@ bridge:
     temporary_disconnect_notices: false
     # Disable bridge notices entirely
     disable_bridge_notices: false
+    # Should Matrix m.notice-type messages be bridged?
+    bridge_notices: true
```

maybe this makes it clearer which direction is being talked about?

```suggestion
    # Should Matrix m.notice-type messages be bridged to Facebook?
    bridge_matrix_notices: true
```

---

You are right, I applied the suggested changes